omas/tests/test_omas_examples.py

```diff
@@ -97,6 +97,12 @@ def test_uncertain(self):
 
     @unittest.skipIf(failed_OMFIT, str(failed_OMFIT))
     def test_plot_g_s_2_ip(self):
+        # on 3.7 this test raises:
+        # ValueError: Number of rows must be a positive integer, not 7.0
+        # It seems like this is caused by a bug in omfit_classes where a float
+        # instead of int is passed to plot
```

Comment:
@orso82 Any idea if this has been fixed on the OMFIT side?  I assume we should just cast it?

Reply:
Probably this needs to be fixed on the OMFIT side. Could you please take care of it @kalling ?

Reply:
@orso82 Alright.
@hassec Would you mind sending me (or pointing me to) a trace of the error that occurs if you don't skip this test?

Reply:
@kalling no problem. It's in `omfit_gapy.py` which has the following lines:
```
        cplot = max([np.floor(np.sqrt(nplot)), 1.0])
        rplot = np.ceil(nplot / cplot)
```
which should probably we wrapped in `int(...)` 

Corresponding ci run with the error [is here](https://github.com/hassec/omas/actions/runs/8119419504/job/22195279200#step:10:948)

Reply:
`cplot` is used in other calculations later that look like they may need it to be a float, so I submitted a change to `auto_merge` that stores the versions passed in to subplot as `plot_cols`  and `plot_rows` instead of doing the cast at assignment, just to be safe.  Re-ran the tests locally and it took care of the issue.

Reply:
@kalling @orso82 What's your preference here? Keep this for now or remove and wait for a new `omfit_classes` release with the fix?

Reply:
Unless it's urgent that this get through, I'd say wait.

Reply:
@kalling  What's the timeline to get a new `omfit_classes` release onto pypi?

Reply:
Assuming the tests pass, it is a weekly process.

Reply:
@kalling are you sure? The [latest version on pypi](https://pypi.org/project/omfit-classes/#history) is from Nov 2023

Reply:
@kalling @orso82 How would you feel about merging as is, and I can follow up as soon as the fix on `omfit_classes` is released on PyPi? Or if it's coming in the next days I'm also happy to wait. 

There isn't really a strong urgency, but I'd like to avoid that the PR sits too long, goes stale and gets forgotten.

Reply:
@hassec Interesting, I wonder if the 2 factor auth introduced messed up the automation for uploading to `pypi`. It should currently be at `3.2024.09`
I'll take a look.

Reply:
Ok, I fixed the `pypi` uploading issue for `omfit-classes` and `3.2024.9.2` was uploaded. Hopefully now it'll actually be weekly again :) (Assuming tests pass)
Thanks for pointing that out.

Reply:
Thanks @kalling I'll have an update PR for OMAS ready asap to remove the workaround :)